source/templates/application_template.html.md.erb

```diff
@@ -16,37 +16,63 @@ This application is retired.
 
 <% else %>
 
-<% unless application.topics.nil? %>
-GitHub topics: <%= application.topics.map { |topic| link_to(topic, "https://github.com/search?q=topic:#{topic}+org:alphagov") }.to_sentence %>
+<% if application.team || application.dependencies_team %>
+## Ownership
+
+<% if application.team == application.dependencies_team %>
+**<%= application.team %>** owns the application and is responsible for updating its dependencies.
+<% elsif application.team && application.dependencies_team %>
+**<%= application.team %>** owns the application, but **<%= application.dependencies_team %>** is responsible for updating its dependencies.
+<% elsif application.team %>
+**<%= application.team %>** owns the application.
+<% elsif application.dependencies_team %>
+**<%= application.dependencies_team %>** is responsible for updating its dependencies.
+<% end %>
 <% end %>
 
-<% if application.team %>
-
-### Ownership
+## Hosting
 
-Owned by team **<%= application.team %>**.
+<% if application.hosting_name == "None" %>
+There is no production version of this application yet.
+<% else %>
+The production version of this application is hosted on **<%= application.hosting_name %>**.
 <% end %>
 
-<% if application.dependencies_team %>
+<% case application.production_hosted_on %>
+<% when  "carrenza" %>
+### SSH Access (Carrenza & AWS)
 
-### Dependencies Team
+This application lives on `<%= application.carrenza_machine %>` machines in Carrenza.
 
-The team responsible for updating dependencies is **<%= application.dependencies_team %>**.
-<% end %>
+```bash
+gds govuk connect ssh -e staging carrenza/<%= application.carrenza_machine %>
+gds govuk connect ssh -e production carrenza/<%= application.carrenza_machine %>
+```
 
-### Hosting
+This application lives on `<%= application.aws_puppet_class %>` machines on the integration environment in AWS.
 
-<% if application.hosting_name == "None" %>
-There is no production version of this application yet.
-<% else %>
-The production version of this application is hosted on **<%= application.hosting_name %>**.
+```bash
+gds govuk connect ssh -e integration <%= application.aws_puppet_class %>
+```
+
+<% when  "aws" %>
+### SSH Access (AWS)
+
+```bash
+gds govuk connect ssh -e integration <%= application.aws_puppet_class %>
+gds govuk connect ssh -e staging aws/<%= application.aws_puppet_class %>
+gds govuk connect ssh -e production aws/<%= application.aws_puppet_class %>
+```
 <% end %>
 
-### Links
+<% if application.can_run_rake_tasks_in_jenkins? %>
+## Run a rake task
 
-<ul>
-  <li><%= link_to "GitHub repo", application.repo_url %></li>
+<%= RunRakeTask.links(application) %>
+<% end %>
 
+<h2>Links</h2><!--FIXME - markdown not working-->
```

Comment:
I had a play with this, and found that we can fix it by [putting the `<ul>`s on newlines](https://github.com/alphagov/govuk-developer-docs/blob/master/app/run_rake_task.rb#L19). You'll need to restart the app to reload the code.

Reply:
Thanks for investigating, it was driving me crazy! I'll see if I can incorporate your suggestion into the commit.

Reply:
Hi Ben, I may be misunderstanding your suggestion, but every I attempt I make is _not_ converting the markdown 😢 

```md


##Links


<ul>


  <% if application.sentry_url %>
```

![Screenshot 2020-05-21 at 10 42 24](https://user-images.githubusercontent.com/5111927/82546279-c8189f80-9b4f-11ea-8d41-c64bb6c09601.png)

Reply:
So I'm saying you need to reformat [this specific line](https://github.com/alphagov/govuk-developer-docs/blob/master/app/run_rake_task.rb#L19) with new lines. That worked for me.

Reply:
Thanks Ben - I've made you a co-author!